staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.pb.go

```diff
@@ -6964,6 +6964,395 @@ func (m *ReopenContainerLogResponse) XXX_DiscardUnknown() {
 
 var xxx_messageInfo_ReopenContainerLogResponse proto.InternalMessageInfo
 
+// Common options used for checkpointing and restoring.
+type CheckpointRestoreOptions struct {
```

Comment:
How about adding an image directory option for a custom checkpoint storage directory that checkpoint and restore could use to save the checkpoint and restore from the checkpoint respectively?